client/components/Layout.js

```diff
@@ -4,10 +4,10 @@ import Header from "./Header";
 
 export default function Layout({ children }) {
   return (
-    <div className="min-h-screen min-w-screen  bg-primary">
-      <div className="flex flex-col max-w-6xl mx-auto min-h-screen ">
+    <div className="min-h-screen min-w-screen bg-primary">
+      <div className="relative flex flex-col max-w-[90rem] mx-auto min-h-screen font-mono">
         <Header />
-        <div className="flex flex-grow">{children}</div>
```

Comment:
Maybe we should keep this or implement another solution to keep the footer to the bottom of the page?